src/multiselect-combo-box.js

```diff
@@ -1,6 +1,8 @@
-import '@vaadin/input-container/src/vaadin-input-container.js';
+import '@polymer/polymer/lib/elements/dom-repeat.js';
+import '@vaadin/polymer-legacy-adapter/template-renderer.js';
```

Comment:
I had to add this import to avoid warning about `<template>` usage which is caused by `dom-repeat` 🙈

Reply:
That's alright 😄